spark-common/src/main/java/com/k15t/spark/atlassian/AtlassianAppServlet.java

```diff
@@ -19,28 +18,24 @@
 import javax.servlet.http.HttpServletResponse;
 
 import java.io.IOException;
-import java.util.Collections;
 import java.util.Locale;
-import java.util.Map;
 import java.util.Objects;
 
 
 public abstract class AtlassianAppServlet extends AppServlet {
 
     private final LoginUriProvider loginUriProvider;
     private final UserManager userManager;
-    private final TemplateRenderer templateRenderer;
     private final LocaleResolver localeResolver;
     private final ApplicationProperties applicationProperties;
     private final long pluginModifiedTimestamp;
 
 
-    protected AtlassianAppServlet(LoginUriProvider loginUriProvider, UserManager userManager, TemplateRenderer templateRenderer,
-            LocaleResolver localeResolver, ApplicationProperties applicationProperties, ApplicationContext applicationContext) {
+    protected AtlassianAppServlet(LoginUriProvider loginUriProvider, UserManager userManager, LocaleResolver localeResolver,
```

Comment:
Due to this change, I needed to adapt the constructor in Backbone as well - but no real impacting change here.